changelog/@unreleased/pr-4110.v2.yml

```diff
@@ -0,0 +1,5 @@
+type: improvement
+improvement:
+  description: Correctly rate limit bad targeted sweep clients.
```

Comment:
I think we should say what `Correctly` means, it's a bit unclear exactly what number we're aiming for. I gather that for each strategy, we want to allow 2 requests per second for the smaller of `(shards, threads)` (to kind-of simulate the old behaviour)? Though not sure if `threads` is intended to be the sum across all nodes (which is a closer simulation of old behaviour) or the number on each node (more conservative)